client/lib/directly/test/index.js

```diff
@@ -96,22 +80,20 @@ describe( 'index', () => {
 		const name = 'Richie Rich';
 		const email = 'richie@richenterprises.biz';
 
-		it( 'does nothing if Directly hasn\'t been initialized', () => {
+		it( 'initializes Directly if it hasn\'t already been initialized', () => {
```

Comment:
oh wow - @gwwar why the constraint? is it just out of date or do we have a reason for not allowing backtics?